### PR TITLE
Task swe 1499 rewrite alphabet primary screen

### PR DESCRIPTION
**Pull request by whoiam460**

### CODE REVIEW

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 1]
```diff
@@ -22,20 +22,22 @@ const ContentScreen = (props) => {
...
-          subtitle={subtitle}
-          chapter={chapter}
-          domainName={domainName}
-          description={description}
-        />
-        <ContentScreenContent content={content} />
-      </ContentScreenContainer>
-      <Footer pageNumber={pageNumber} color={footerColor} />
-    </ContentScreenStyled>
+    <>
```
Comment (olenashevchuk) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:25]: useless <>

File: src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js [hunk 2]
```diff
@@ -3,5 +3,6 @@ import styled from 'styled-components'
 
 const ContentScreenContainerStyled = styled(Container)`
   flex: 1 0 auto;
+  margin-left: var(--brandbook-module-content-screen-margin-left);
```
Comment (olenashevchuk) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js:6]: !!!!

File: src/app/screens/Typography/TypographyContent/SecondaryScreen.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      subtitle="Semibold / Medium / Regular"
+      chapter="Secondary"
+      domainName="Typo."
+      // TODO get font family from  vars
+      content={
+        <Box mt={4}>
+          <Alphabet />
+        </Box>
+      }
+      pageNumber={'09'}
```
Comment (olenashevchuk) [inline src/app/screens/Typography/TypographyContent/SecondaryScreen.js:19]: delete {}

File: src/styles/vars/contentScreen.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+:root {
+
+  
+    --brandbook-module-content-screen-margin-left: 96px;
```
Comment (olenashevchuk) [inline src/styles/vars/contentScreen.css:4]: !!!!!!!!!!!

File: src/app/screens/Typography/TypographyContent/PrimaryScreen.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      subtitle="Regular"
+      chapter="Primary"
+      domainName="Typo."
+      // TODO get font family from  vars
+      content={
+        <Box mt={3}>
+          <Alphabet ff="primary" />
+        </Box>
+      }
+      pageNumber={'09'}
```
Comment (Slavka406) [inline src/app/screens/Typography/TypographyContent/PrimaryScreen.js:19]: ?

File: src/app/screens/Typography/TypographyContent/PrimaryScreen.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    <ContentScreen
+      breadcrumbs={['{brandName} — Guidelines']}
+      title="Plus Jakarta Sans"
+      subtitle="Regular"
+      chapter="Primary"
+      domainName="Typo."
+      // TODO get font family from  vars
+      content={
+        <Box mt={3}>
+          <Alphabet ff="primary" />
```
Comment (Slavka406) [inline src/app/screens/Typography/TypographyContent/PrimaryScreen.js:16]: ???

File: src/app/components/elements/Alphabet/Alphabet.js [hunk 7]
```diff
@@ -1,21 +1,21 @@
+import { ALPHABET } from 'app/constants'
 import { AlphabetStyled } from './Alphabet.styled'
 import PropTypes from 'prop-types'
 import { Text } from '~/app/components'
 
 const Alphabet = (props) => {
-  const { alphabet, fontFamily } = props
+  const { fontFamily } = props
```
Comment (Slavka406) [inline src/app/components/elements/Alphabet/Alphabet.js:7]: ????

File: src/app/components/elements/Alphabet/Alphabet.js [hunk 8]
```diff
@@ -1,21 +1,21 @@
...
 
   return (
     <AlphabetStyled>
       <Text variant="brand-alphabet" fm={fontFamily}>
```
Comment (Slavka406) [inline src/app/components/elements/Alphabet/Alphabet.js:11]: ???

**Review by Slavka406 — CHANGES_REQUESTED** [review]

File: src/app/screens/Typography/TypographyContent/PrimaryScreen.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import { Alphabet } from 'app/components'
+import { Box } from 'components'
+import { ContentScreen } from 'components'
+
+const PrimaryScreen = () => {
```
Comment (Olexandr25) [inline src/app/screens/Typography/TypographyContent/PrimaryScreen.js:5]: write comment about this component
what return and for what it

**Pull request closed without merging**